js/layout-javascript/small-card-code.js

```diff
@@ -132,9 +132,13 @@ DynamicList.prototype.attachObservers = function() {
     }
   });
 
-  $(window).resize(function() {
-    _this.Utils.DOM.adjustAddButtonPosition(_this);
-  });
+  $(window)
+    .resize(function() {
+      _this.Utils.DOM.adjustAddButtonPosition(_this);
+    })
+    .on('show.bs.dropdown', function() {
```

Comment:
What if the LFD detail view also has Bootstrap dropdowns implemented? It seems that we only want to implement this for the sort dropdown, not all dropdowns.

Reply:
Also, please add inline comments for clarification

Reply:
If LFD has also Bootstrap dropdown they will also become cut by accordion, that's why I think we should implement this not just for sort dropdowns.
Added inline comments for clarification.

Reply:
Also, I have solution for accordion heading widget with only styles 
![image](https://user-images.githubusercontent.com/74735207/106592330-0e842200-6558-11eb-871c-9b715914cec6.png)

Reply:
@ivandevupp Thanks. That makes sense. In this case your solution has a few problems.

The `show.bs.dropdown` and `hide.bs.dropdown` event listeners should be applied to `_this.$container` as the `attachObservers()` function already has. This is so that:

1. The behaviour is restricted to LFDs
2. The behaviour is restricted to the specific instance of LFD

When you apply the `overflow` style, you shouldn't use `$('[data-collapse-id]').find('div.in')` as this looks for all collapses in the entire page. Instead, look for the parent accordion that contains the LFD, e.g. something like  `_this.$container.parents('[data-collapse-id]').find('div.in')`.

Reply:
I don't know why, but at the first time I couldn't attach observer to container, it didn't fire, that's why I attached it to window, but now it works.
Fixed for SmallCard/SimpleList/NewsFeed.